src/ruby/puppetserver-lib/puppet/server/http_client.rb

```diff
@@ -44,17 +44,18 @@ def self.settings
     @settings ||= {}
   end
 
-  def initialize(server, port, options = {})
+  def initialize(options = {})
     options = OPTION_DEFAULTS.merge(options)
```

Comment:
The new client defaults the `X-Puppet-Version` and `User-Agent` headers based on `Puppet.version` and `Puppet[:http_user_agent]`. Does puppetserver/clj-http-client also do that (or something similar)?

Reply:
No, I don't think so. But we can add that here if we want them to be the same.

Reply:
Just realizing that I'm pretty sure this merge does nothing. `options` is a local var that we throw away. I'll need to look into how to properly pass that other configuration.